eth/blockwatch/block_watcher_test.go

```diff
@@ -246,6 +274,31 @@ func TestGetMissedEventsToBackfillNoneMissed(t *testing.T) {
 	assert.Equal(t, big.NewInt(5), headers[0].Number)
 }
 
+func TestGetMissedEventsToBackfill_NOOP(t *testing.T) {
+	// No last retained block
+	// No config.BackfillStartBlock
+
+	fakeClient, err := newFakeClient("testdata/fake_client_basic_fixture.json")
+	require.NoError(t, err)
+
+	store := &stubMiniHeaderStore{}
+
+	config.Store = store
+	config.Client = fakeClient
+	watcher := New(config)
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	events, err := watcher.getMissedEventsToBackfill(ctx)
+	require.NoError(t, err)
+	assert.Len(t, events, 0)
+
+	// Check that block 5 is still in the DB
```

Comment:
This comment doesn't seem relevant to this test?

Reply:
fixed in be50d19d